doc/api/crypto.md

```diff
@@ -944,6 +944,15 @@ The `output_format` can specify one of `'latin1'`, `'hex'` or `'base64'`. If
 `output_format` is provided a string is returned; otherwise a [`Buffer`][] is
 returned.
 
+The optional `options` argument is an object which specifies additional
+cryptographic parameters. Currently, the following options are supported:
+
+* `padding` : {String} - RSA padding, either `'pkcs1'` for RSASSA-PKCS1-v1_5
+  (default) or `'pss'` for RSASSA-PSS
+* `saltLength` : {number} - salt length for RSASSA-PSS. If this is set to `-1`,
+  the salt length will be set to the digest size. If this is set to `-2`
+  (default), the salt length will be set to the maximum permissible value.
```

Comment:
It might be nice to have these two values exposed as named constants on `crypto.constants`… what do you think?